frontend/src/App.css

```diff
@@ -11,6 +11,52 @@
   width: 80%;
 }
 
+.cancel-delete-btn {
+  background: none;
+  line-height: 1.5;
+  font-size: 100%;
+  display: inline-block;
+  cursor: pointer;
+  margin-left: 5px;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: bold;
+  padding: 6px 15px;
+  text-decoration: none;
+  text-shadow: 1px 1px 0 rgba(0, 0, 0, 0.5);
+  box-shadow: inset 0 1px 0 0 #23b306;
+  background-image: -webkit-linear-gradient(
+    top,
+    rgba(255, 255, 255, 0.3),
+    rgba(255, 255, 255, 0)
+  );
+  background-color: #fe1a00;
+  border: 1px solid #d83526;
+}
+
+.confirm-delete-btn {
+  background: none;
+  line-height: 1.5;
+  `font-size: 100%;
+  display: inline-block;
+  cursor: pointer;
+  margin-left: 5px;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: bold;
+  padding: 6px 15px;
+  text-decoration: none;
+  text-shadow: 1px 1px 0 rgba(0, 0, 0, 0.5);
+  box-shadow: inset 0 1px 0 0 rgba(255, 255, 255, 0.5);
+  background-image: -webkit-linear-gradient(
+    top,
+    rgba(255, 255, 255, 0.3),
+    rgba(255, 255, 255, 0)
+  );
+  background-color: #23b306;
+  border: 1px solid #3b7808;
+}
+
 .noShow {
   display: none !important;
```

Comment:
Are you sure you need that important declaration

Stephanie Rewis
> Using !important in your CSS usually means you’re narcissistic & selfish or lazy. Respect the devs to come…

https://css-tricks.com/when-using-important-is-the-right-choice/

Reply:
It is needed for hiding toolbar in ReactMD, which is external library. Cannot do it any other way.